third_party/blink/renderer/core/farbling/brave_session_cache.cc

```diff
@@ -263,8 +245,20 @@ void BraveSessionCache::Init() {
 }
 
 void BraveSessionCache::FarbleAudioChannel(float* dst, size_t count) {
-  if (audio_farbling_helper_)
-    audio_farbling_helper_->FarbleAudioChannel(dst, count);
+  if (!audio_farbling_helper_) {
+    const auto audio_farbling_level =
+        GetBraveFarblingLevel(ContentSettingsType::BRAVE_WEBCOMPAT_AUDIO);
+    if (audio_farbling_level == BraveFarblingLevel::OFF) {
+      return;
```

Comment:
you're still requesting `GetBraveFarblingLevel` each time `FarbleAudioChannel` is called if farbling is `OFF`. This is bad and should be fixed.

Reply:
`GetBraveFarblingLevel` caches its result in a map, so there is no expensive rpc after the first time it is called. So we are leaving it as is, but I added a comment in the code explaining this.